server/routes.go

```diff
@@ -720,9 +720,49 @@ func GetModelInfo(req api.ShowRequest) (*api.ShowResponse, error) {
 	fmt.Fprint(&sb, model.String())
 	resp.Modelfile = sb.String()
 
+	ggmlData, err := getGGMLData(model)
+	if err != nil {
+		return nil, err
+	}
+	resp.ModelInfo = ggmlData
+
 	return resp, nil
 }
 
+func getGGMLData(model *Model) ([]byte, error) {
+	f, err := os.Open(model.ModelPath)
+	if err != nil {
+		return nil, err
+	}
+
+	ggml, _, err := llm.DecodeGGML(f)
+	if err != nil {
+		return nil, err
+	}
+
+	f.Close()
+
+	kv := ggml.KV()
+
+	for k := range kv {
+		switch v := kv[k].(type) {
+		case []interface{}:
+			if len(v) > 5 {
+				kv[k] = []string{}
+			}
+		}
+	}
+
+	kv["embedding_model"] = model.IsEmbedding()
```

Comment:
I'm not sure about adding this as a KV since it's not part of the model itself. Ideally this can be captured as a top-level `capabilities`